hookman/hooks.py

```diff
@@ -115,25 +115,38 @@ def remove_plugin(self, plugin_name: str):
                 shutil.rmtree(plugin.yaml_location.parents[1])
                 break
 
-    def get_plugins_available(self) -> Optional[List[PluginInfo]]:
+    def get_plugins_available(self, ignored_plugins: List[str]=None) -> Optional[List[PluginInfo]]:
         """
         Return a list of :ref:`plugin-info-api-section` that are available on ``plugins_dirs``
 
+        Optionally you can pass a list of plugins that should be ignored.
+
         The :ref:`plugin-info-api-section` is a object that holds all information related to the plugin.
         """
         plugin_config_files = hookman_utils.find_config_files(self.plugins_dirs)
-        return [PluginInfo(plugin_file, self.hooks_available) for plugin_file in plugin_config_files]
```

Comment:
You could just add `if plugin_info.name not in ignored_plugins` to this list comprehension:

```python
return [PluginInfo(x, self.hooks_available) for xin plugin_config_files 
        if x.name not in ignored_plugins]
```

Also change the signature to accept a `Sequence[str]=()`, so you don't even need to handle `None`. 😉